package.json

```
@@ -33,10 +34,11 @@
     "prepack": "sed -ri 's/\\[!([A-Z]+)\\]/**\\1**:/g' README.md",
     "test": "run-s lint test:all",
     "test:all": "c8 run-s test:unit test:integration",
-    "test:integration": "run-s test:integration:mocha test:integration:playwright test:integration:web-test-runner && mocha test/integration/report-validation.test.js",
+    "test:integration": "run-s test:integration:mocha test:integration:playwright test:integration:web-test-runner test:integration:webdriverio && mocha test/integration/report-validation.test.js",
```

Script `test:integration:webdriver-io` please.

```suggestion
    "test:integration": "run-s test:integration:mocha test:integration:playwright test:integration:web-test-runner test:integration:webdriver-io && mocha test/integration/report-validation.test.js",
```